python2/koans/about_scoring_project.py

```diff
@@ -34,8 +34,21 @@
 # Your goal is to write the score method.
 
 def score(dice):
+    final_score = 0
+    repeated_nums = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+    for number in dice:
+        repeated_nums[number]=repeated_nums[number] + 1
+    for i in range(0,10):
+        aux1 = int (repeated_nums[i] / 3)
```

Comment:
Tip: Python has two division operators: `/` for floating division, and `//` for int division. 

```python
aux1 = int (repeated_nums[i] / 3)
```

is the same as 

```python
aux1 = repeated_nums[i] // 3
```